locales/puppetserver.pot

```diff
@@ -251,14 +251,6 @@ msgstr ""
 msgid "Removed certificate request for {0} at ''{1}''"
 msgstr ""
 
-#: src/clj/puppetlabs/puppetserver/certificate_authority.clj
-msgid "Revoked {0} certificate with serial {1}"
-msgstr ""
-
-#: src/clj/puppetlabs/puppetserver/certificate_authority.clj
-msgid "Infra node certificate being revoked; publishing updated infra CRL"
-msgstr ""
-
```

Comment:
why was this one removed? looks like that message is still there?

Reply:
Bad conflict resolution, I'll put it back. Though it may not be exactly right anymore...

Reply:
This should get regenerated in Jenkins with the new messages.